doc/arch/adr-005-api-client-app-auth.md

```diff
@@ -0,0 +1,121 @@
+# Geoprocessing API: Only Allow Anonymous Requests From The Client App
+
+## Context
+
+We're creating a [proper, publicly documented API](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/geoprocessing_api/views.py) around our analyze and 
+RWD endpoints: the [Geoprocessing API.](https://github.com/WikiWatershed/model-my-watershed/blob/develop/doc/arch/adr-004-geoprocessing-api.md) The analyze and RWD endpoints have
+have allow-any access; we'd now like to secure them via token authentication
```

Comment:
"have had", maybe?  Or even, "have always had"?

Reply:
There are a terrifying number of repeated words...I think this was "have always had" before I mangled it.